app/src/main/java/com/sun/mangareader01/ui/main/MainActivity.kt

```diff
@@ -108,6 +107,21 @@ class MainActivity : FragmentActivity(),
         return true
     }
 
+    override fun <T> onClick(item: T): Unit? = when (item) {
```

Comment:
a nghĩ e nên tạo ra nhiều callback sẽ tốt hơn là tạo 1 callback xong check is ntn